rspec_files/base_decorator_rspec.rb

```diff
@@ -0,0 +1,12 @@
+require './base_decorator'
+
+describe Decorator do
+  context 'Instance of base decorator' do
+    before :each do
+      @base_decorator = Decorator.new('Peet')
+    end
+    it 'return a new base decorator' do
+      expect(@base_decorator).to be_instance_of Decorator
+    end
+  end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_unit_tests.md#:~:text=all%20classes%20that%20you%20created%20while%20implementing%20the%20Decorator%20design%20pattern), Please also add tests for all classes that you created while implementing the Decorator design pattern like `capitalized_decorator.rb` and `trimmer_decorator.rb`.